src/test/scala/com/cognite/sdk/scala/common/WritableBehaviors.scala

```diff
@@ -185,6 +185,21 @@ trait WritableBehaviors extends Matchers { this: FlatSpec =>
       updatable.deleteByIds(createdItems.map(_.id))
     }
 
+  def deletableWithIgnoreUnkownIds[R <: WithId[Long], W, PrimitiveId](
+    writable: Create[R, W, Id] with DeleteByIdsWithIgnoreUnknownIds[Id, Long] with RetrieveByIds[R, Id],
+    readExamples: Seq[R],
+    idsThatDoNotExist: Seq[Long]
+  )(implicit t: Transformer[R, W]): Unit = {
+    it should "support ignoring unknown ids on delete" in {
+      val createdItems: Id[Seq[R]] = writable.createFromRead(readExamples)
+      val createdIds = createdItems.map(_.id)
+      val createdAndSomeNonExistingIds = createdIds ++ idsThatDoNotExist
+      writable.deleteByIds(createdAndSomeNonExistingIds, ignoreUnknownIds = true)
```

Comment:
Try deleting only items that do not exist as well.

Change the case of `ignoreUnknownIds = false` as well and make sure it throws an exception.